private/buf/bufprotopluginexec/version.go

```diff
@@ -120,6 +121,23 @@ func getFeatureProto3OptionalSupported(version *pluginpb.Version) bool {
 	return true
 }
 
+// Should I notify that I am OK with editions (and, if so, which ones)?
+func getFeatureEditionsSupported(version *pluginpb.Version) (supported bool, min, max descriptorpb.Edition) {
+	if version.GetMajor() > 5 || (version.GetMajor() == 5 && version.GetMinor() >= 27) {
```

Comment:
When testing locally with protoc 27.1, I saw the major version set to 27 and the minor version set to 1.

Reply:
Oof, you're right. I was also looking at the output of `buf alpha protoc --version`, which prints 5.27 :(

I guess we may want to change that, too.

Reply:
I've updated the logic to handle both the `buf alpha protoc` output and actual `protoc` output and updated the tests accordingly.

I also changed it so that `buf alpha protoc` going forward will omit the C++ runtime major version. So with this PR it would print `27.0-buf` (compared to actual `protoc` printing `libprotoc 27.0`). Previously, it printed `5.27.0-buf`.